src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      return firstNumber + secondNumber;
+    case '-':
+      return firstNumber - secondNumber;
+    case 'X':
+      return firstNumber * secondNumber;
+    case '÷':
+      if (secondNumber === '0') {
+        return '0';
+      }
+      return firstNumber / secondNumber;
+    case '%':
+      return firstNumber * Big(0.01);
+    case '+/-':
+      return firstNumber * Big(-1);
```

Comment:
Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case division will be -> `firstNumber.minus(secondNumber)`